vm/actor/src/builtin/paych/mod.rs

```diff
@@ -224,18 +223,14 @@ impl Actor {
             // 4. check operation validity
             let new_send_balance = st.to_send.clone() + balance_delta;
```

Comment:
could you change this to taking references instead of cloning?

Reply:
ah, ya, glossed over that one, assumed way it was setup was necessary